vault/rollback.go

```diff
@@ -222,13 +247,14 @@ func (m *RollbackManager) startOrLookupRollback(ctx context.Context, fullPath st
 		// we already have the inflight lock, so we can't grab it here
 		m.finishRollback(rs, errors.New("rollback manager is stopped"), fullPath, false)
 	default:
-		m.runner.Submit(func() {
-			m.attemptRollback(ctx, fullPath, rs, grabStatelock)
-			select {
-			case m.rollbacksDoneCh <- struct{}{}:
-			default:
-			}
-		})
+		job := &rollbackJob{
+			ctx:           ctx,
+			fullPath:      fullPath,
+			rs:            rs,
+			grabStatelock: grabStatelock,
+			manager:       m,
+		}
+		m.jobManager.AddJob(job, fullPath)
```

Comment:
@satoqz @wslabosz-reply We'll have to keep an eye on this later. I think what this does is places each rollback job in a separate queue; we might want to have one per-namespace or similar, to avoid starving other namespaces when a single namespace has a lot of mounts.

But we can deal with that later.